src/main/java/net/openhft/chronicle/wire/internal/UnknownClassLookup.java

```diff
@@ -0,0 +1,55 @@
+package net.openhft.chronicle.wire.internal;
+
+import net.openhft.chronicle.core.Jvm;
+import net.openhft.chronicle.core.OS;
+import net.openhft.chronicle.core.pool.ClassLookup;
+import net.openhft.chronicle.core.util.ClassNotFoundRuntimeException;
+import net.openhft.compiler.CachedCompiler;
+
+import java.io.File;
+
+public class UnknownClassLookup implements ClassLookup {
+    public static final CachedCompiler CACHED_COMPILER =
+        new CachedCompiler(Jvm.isDebug() ? new File(OS.getTarget(), "generated-test-sources") : null, null);
```

Comment:
This class should be moved to test sources or target has to be adjusted respectively.